score.cwl

```diff
@@ -31,6 +31,11 @@ inputs:
     type: string?
     inputBinding:
       prefix: -q
+  
+  - id: submission_number
+    type: int?
```

Comment:
Optional parameter?  What is the seed when the submission number isn't given?

Reply:
Hmm... good point.  The model originally defaulted to 0, but I have since removed that. 

Will probably remove this as optional.